imu_eskf_node:
  ros__parameters:
    use_sim_time: false
    clock_type: "ros"
    imu_topic: "/imu/data_newIMU"
    mag_topic: "/mag_newIMU"
    use_magnetometer: true
    noise_filter: true
    filter_gyro: false
    fc: 25.0
    fs: 100.0
    mag_bias_x: 0.0
    mag_bias_y: 0.0
    mag_bias_z: 0.0
    acc_noise: 0.0008
    mag_noise: 0.00000001
    gyro_noise: 0.00008
    gyro_bias_noise: 0.0000000007
    theta_noise: 0.00001
    wb_noise: 0.0000000001
    magnetic_reference_x: 29.14
    magnetic_reference_y: -4.46
    magnetic_reference_z: 45.00
    coordinate_frame: "NED"